src/sentry/ownership/grammar.py

```diff
@@ -386,25 +390,38 @@ def parse_code_owners(data: str) -> Tuple[List[str], List[str], List[str]]:
     return teams, usernames, emails
 
 
-def convert_codeowners_syntax(data, associations, code_mapping):
+def convert_codeowners_syntax(codeowners, associations, code_mapping):
     """Converts CODEOWNERS text into IssueOwner syntax
-    data: CODEOWNERS text
+    codeowners: CODEOWNERS text
     associations: dict of {externalName: sentryName}
     code_mapping: RepositoryProjectPathConfig object
     """
 
     result = ""
 
-    for rule in data.splitlines():
+    for rule in codeowners.splitlines():
         if rule.startswith("#") or not len(rule):
             # We want to preserve comments from CODEOWNERS
             result += f"{rule}\n"
             continue
 
-        path, *codeowners = rule.split()
+        path, *code_owners = (x.strip() for x in rule.split())
+        # Escape invalid rules
```

Comment:
by rule here do we just mean the path? if so I think we should say path instead

Reply:
and maybe we can link to https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners#syntax-exceptions